Samples~/Polybrush.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: 63867c13d780e894bad0da0b1b44538a
+folderAsset: yes
```

Comment:
are these polybrush files meant to be included?

Reply:
no indeed... removing these